recommenders/models/sar/sar_singlenode.py

```diff
@@ -157,7 +161,7 @@ def compute_time_decay(self, df, decay_column):
         # group time decayed ratings by user-item and take the sum as the user-item affinity
         return df.groupby([self.col_user, self.col_item]).sum().reset_index()
 
-    def compute_cooccurrence_matrix(self, df):
+    def compute_cooccurrence_matrix(self, df, reverse=False):
```

Comment:
I agree with your critique. In hindsight, reverse was not a clear name for this. Personally, I prefer booleans because they leave less room for error.